test/src/main/java/org/jvnet/hudson/test/recipes/WithPlugin.java

```diff
@@ -50,9 +50,9 @@
 @Retention(RUNTIME)
 public @interface WithPlugin {
     /**
-     * Name of the plugin.
+     * Whitespace separated list of plugin names.
```

Comment:
imho whitespace is bad delimiter, i already had experience with AD where whitespace is valid. Now i use only CSV, wouldn't it better to use CSV?